src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs

```diff
@@ -296,10 +346,9 @@ public IList<string> Aud
             {
                 if (_audiences == null)
                 {
-                    List<string> tmp = GetListOfClaims(JwtRegisteredClaimNames.Aud);
+                    List<string> tmp = new (GetListOfClaims(JwtRegisteredClaimNames.Aud));
```

Comment:
I don't understand, GetListOfClaims already returns a list of strings, what is the `new` for?